src/app/modules/submission/email-submission/email-submission.service.ts

```diff
@@ -254,3 +256,29 @@ export const saveSubmissionMetadata = (
     },
   )
 }
+
+export const sendSubmissionToAdmin = (
+  adminEmailParams: Parameters<typeof MailService['sendSubmissionToAdmin']>[0],
+): ResultAsync<true, SendAdminEmailError> => {
+  const errorLogParams = {
+    message: 'Error sending submission to admin',
+    meta: {
+      action: 'sendSubmissionToAdmin',
+      submissionId: adminEmailParams.submission.id,
+      formId: adminEmailParams.form._id,
+    },
+  }
+  return ResultAsync.fromPromise(
+    MailService.sendSubmissionToAdmin(adminEmailParams),
+    (error) => {
+      logger.error({ ...errorLogParams, error })
+      return new SendAdminEmailError()
+    },
+  ).andThen((result) => {
+    if (!result) {
```

Comment:
I don't think `sendSubmissionToAdmin` can return false right? It's `true` or error. I guess we can change the signature to `Promise<true>` instead.

Reply:
LOL ya sorry I was lazy to change the signature

Reply:
ok changed!